src/commons/TextInput.tsx

```diff
@@ -1,10 +1,12 @@
-import React, { ComponentPropsWithoutRef, FC } from "react";
+import React, { ComponentPropsWithoutRef, FC, useState } from "react";
+import { AiOutlineQuestionCircle } from "react-icons/ai";
 
 interface TextInputProps extends ComponentPropsWithoutRef<"input"> {
   name: string;
   label?: string;
   error?: string;
   helper?: string;
+  toolkit?: string;
```

Comment:
Sería `tooltip`, pequeño error de typo 😅
```suggestion
  tooltip?: string;
```

Reply:
Lo mismo en todas las coincidencias de abajo